benchmarks/bm_runner.py

```diff
@@ -48,27 +48,27 @@ def _prep_data_gen_env() -> None:
     """
 
     root_dir = Path(__file__).parents[1]
-    python_version = "3.10"
+    python_version = "310"
     data_gen_var = "DATA_GEN_PYTHON"
     if data_gen_var in environ:
         print("Using existing data generation environment.")
     else:
         print("Setting up the data generation environment ...")
-        # Get Nox to build an environment for the `tests` session, but don't
-        #  run the session. Will re-use a cached environment if appropriate.
+        # Get tox to build an environment. It will re-use a cached environment
+        # if appropriate.
         subprocess.run(
             [
-                "nox",
-                f"--noxfile={root_dir / 'noxfile.py'}",
-                "--session=tests",
-                "--install-only",
-                f"--python={python_version}",
+                "tox",
+                f"-c {root_dir / 'tox.ini'}",
+                f"-e py{python_version}",
```

Comment:
Nice thing about tox is that we get an environment, with a dev installation of iris, for free, because of the definition of:
https://github.com/SciTools/iris/blob/62bcb99d4aff5769f293413a3952ad7b8e580ac2/tox.ini#L5-L11
This allows you to run `tox -e py310` 


Alternatively tox has a `--notests` flag which will create the environment, but not run the commands, aka what nox's `--install-only` was doing